schemas/g-cloud-9.declaration.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
What do you mean when you say "we already did it for all the other repos"?

Only API has schemas in it, surely?